cmake/protobuf-config.cmake.in

```diff
@@ -1,19 +1,30 @@
 # Version info variables
-set(PROTOBUF_VERSION        "@protobuf_VERSION@")
-set(PROTOBUF_VERSION_STRING "@protobuf_VERSION_STRING@")
+set(Protobuf_VERSION        "@protobuf_VERSION@")
```

Comment:
${PACKAGE_FIND_NAME}_VERSION is set automatically by find_package if this module is chosen. This line is unnecessary.